src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -97,6 +97,24 @@ replying to DNS queries.
 
 Nothing extra needs to be done to build `submariner-globalnet` as it is built with the standard Submariner build.
 
+## Prerequisites
+
+In submariner 0.12 or later, Globalnet controller internally uses Kubernetes externalIPs feature,
```

Comment:
> I think we should generally avoid references to Submariner versions in the main docs

@mkimuram This was discussed more on a different PR and the consensus is we *should* use versions, just need to avoid "new". Sorry for the bad feedback! I've sent a PR to clarify this practice in the future: https://github.com/submariner-io/submariner-website/pull/705.

Reply:
Never mind. And thank you for letting me know the decision.